search-service/src/main/kotlin/com/egm/stellio/search/entity/model/Attribute.kt

```diff
@@ -45,7 +45,7 @@ data class Attribute(
     val attributeValueType: AttributeValueType,
     val datasetId: URI? = null,
     val createdAt: ZonedDateTime,
-    val modifiedAt: ZonedDateTime? = null,
+    val modifiedAt: ZonedDateTime? = createdAt, // todo could be non nullable
```

Comment:
this TODO has to be removed. and, yes, since we now automatically set a `modifiedAt` when creating an entity or attribute, it should be non nullable.

Reply:
and what about https://github.com/stellio-hub/stellio-context-broker/blob/develop/search-service/src/main/kotlin/com/egm/stellio/search/entity/model/Entity.kt#L25? (it should also be set for entity, not only attributes)